packages/ssi-service/package.json

```diff
@@ -27,7 +27,7 @@
     "babel-minify-webpack-plugin": "0.3.1",
     "babel-preset-es2015": "6.24.1",
     "clean-webpack-plugin": "3.0.0",
-    "copy-webpack-plugin": "5.1.1",
+    "copy-webpack-plugin": "6.0.1",
```

Comment:
May need to change some code due to the breaking changes.

Reply:
The new version breaks the config. Need to make some changes to `webpack.config.js` in ssi-service when this gets merged.